pages/posts/[post].vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <UProgress animation="carousel" class="my-4" v-if="isLoaderShow" />
+    <template v-else>
+      <div class="flex">
+        <div class="w-20">
+          <p class="font-semibold">Id:</p>
+        </div>
+        <div class="grow">
+          <p>{{ post.id }}</p>
+        </div>
+      </div>
+
+      <div class="flex">
+        <div class="w-20">
+          <p class="font-semibold">User Id:</p>
+        </div>
+        <div class="grow">
+          <p>{{ post.userId }}</p>
+        </div>
+      </div>
+
+      <div class="flex">
+        <div class="w-20">
+          <p class="font-semibold">Title:</p>
+        </div>
+        <div class="grow">
+          <p>{{ post.title }}</p>
+        </div>
+      </div>
+
+      <div class="flex">
+        <div class="w-20">
+          <p class="font-semibold">Body:</p>
+        </div>
+        <div class="grow">
+          <p>{{ post.body }}</p>
+        </div>
+      </div>
+    </template>
+  </div>
+</template>
+<script lang="ts" setup>
+interface Post {
+  id: number,
+  userId: number,
+  title: string,
+  body: string
+}
```

Comment:
Case in point from https://github.com/akshay-lad-credilio/nuxt-demo/pull/1#discussion_r1528180588

Reply:
Resolved